gallery/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components-gallery",
-  "version": "9.5.16",
+  "version": "9.5.17",
```

Comment:
New feature, should be a minor version

Reply:
> New feature, should be a minor version

Check f31d0f9a3f7f40a719569832321352147f4ee34e